{"name": "test_login_fail_form_error[data4]", "status": "passed", "attachments": [{"name": "log", "source": "c629488b-b6a7-4905-84d7-01d675ab774a-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'user': '', 'password': '', 'check': '请输入手机号'}"}], "start": 1567138259945, "stop": 1567138260638, "uuid": "5be1b3c2-3583-45de-911d-7b3735bb2070", "historyId": "30b0e78c37cf50ce179c42e95432c663", "testCaseId": "d9574c1722641d515e9f2f722dc354c5", "fullName": "testcases.test_login6.Test_Login#test_login_fail_form_error", "labels": [{"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_login6"}, {"name": "subSuite", "value": "Test_Login"}, {"name": "host", "value": "liliangdeMacBook-Pro.local"}, {"name": "thread", "value": "68435-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_login6"}]}